db/schema.rb

```diff
@@ -65,7 +87,7 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "first_name"
-    t.string "last_name"
+    t.string "last_name"""
```

Comment:
```suggestion
    t.string "last_name"
```

Reply:
👀 I'm feeling ashamed I did not spot that ><
As of now, and as discussed with Olivier, I will add an extra-time on reviewing carefully the PR before passing it as non-WIP. Sorry for these ones =/